docs/faq/admin.txt

```diff
@@ -94,14 +94,20 @@ presentation by editing the CSS stylesheet and/or associated image files. The
 site is built using semantic HTML and plenty of CSS hooks, so any changes you'd
 like to make should be possible by editing the stylesheet.
 
+.. _admin-browser-support:
+
 What browsers are supported for using the admin?
 ================================================
 
-The admin provides a fully-functional experience to `YUI's A-grade`_ browsers,
-with the notable exception of IE6, which is not supported.
+The admin provides a fully-functional experience to the recent versions of
+modern, web standards compliant browsers. On desktop this means Chrome, Edge,
+Firefox, Opera, Safari, and others.
+
+On smaller, mobile and tablet devices, the admin provides a responsive
+experience for web standards compliant browsers. This includes the major
+browsers on both Android and iOS.
 
-There *may* be minor stylistic differences between supported browsers—for
-example, some browsers may not support rounded corners. These are considered
-acceptable variations in rendering.
+Depending on feature support there *may* be minor stylistic differences between
+browsers. These are considered acceptable variations in rendering.
 
-.. _YUI's A-grade: https://github.com/yui/yui3/wiki/Graded-Browser-Support
+Microsoft's legacy Internet Explorer is not supported.
```

Comment:
Not sure if unsupported browsers deserve a special call out? It seems like a safe assumption that anythign not mentioned isn't supported.

Reply:
I was in two minds about this.

Reply:
OK +1. It's nicer. Thanks for the review both! I shall update now.